care/emr/resources/charge_item/sync_charge_item_costs.py

```
@@ -8,10 +8,17 @@
 
 def calculate_amount(component, quantity, base):
     if component.amount:
+        if component.amount < 0:
+            raise ValidationError("Amount cannot be negative")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)